src/html/event.html

```diff
@@ -0,0 +1,3 @@
+<script type="text/javascript">
+  window.location.href = "https://www.helloasso.com/associations/mixteen/evenements/enseignant-e-s-et-mixteen"
```

Comment:
You dont't need Javascript to do a redirection. See https://en.wikipedia.org/wiki/Meta_refresh

Reply:
de toute façon, autant avoir un vrai raccourcisseur et avoir un truc genre "https://mixteen.org/event?id=XX"
plutôt que cette page qui s'appelle event mais déjà ce jour là on en a 2
Vous en pensez quoi ?

Reply:
@javamind bien vu pour le meta refresh, ça sera mieux.

@titimoby Pas trop fan de mettre un paramètre dans l'url, le but c'est qu'elle soit saisissable facilement car affichée sur papier (pas de caractère spéciaux, nom clair et court).

Reply:
je suis d'accord mais je n'aime pas trop le fait d'avoir un mixteen.org/event  qui soit à usage unique
au minimum mixteen.org/event25mai

Mon idéal serait un raccourciceur d'url à nous, un vrai.
c'est la misère à réaliser ? (pour plus tard, là un /quelquechose bien nommé suffit)

Reply:
@javamind Pour la balise meta, si je la place dans la page elle apparaîtra hors de la balise <head>. En soit ce n'est pas grave, mais en terme de bonne pratique c'est pas top on est d'accord ?